dspace/config/spring/api/core-services.xml

```diff
@@ -82,7 +82,7 @@
     <bean class="org.dspace.license.CreativeCommonsServiceImpl"/>
 
     <bean class="org.dspace.statistics.ElasticSearchLoggerServiceImpl" lazy-init="true"/>
-    <bean class="org.dspace.statistics.SolrLoggerServiceImpl"/>
+    <bean id="solrLoggerService" class="org.dspace.statistics.SolrLoggerServiceImpl"/>
```

Comment:
By using the factory call from above the naming of the bean can be left out here. Else it might cause confusion about which beans are named & which aren't.